src/components/Link/Link.tsx

```diff
@@ -1,50 +1,24 @@
 // TODO: ADS-383 pass down NavLink props e.g. "end"
 
-import { AnchorHTMLAttributes, ForwardedRef, forwardRef, ReactNode, useState, useEffect } from 'react';
-import { NavLink, Link as RouterLink, To } from 'react-router-dom';
-
-export type LinkVariants = '' | 'cta' | 'ghost' | 'mailto' | 'nav' | 'subtle';
-export interface LinkProps extends AnchorHTMLAttributes<HTMLAnchorElement> {
-  children: ReactNode;
-  href?: string;
-  to?: To;
-  variant?: LinkVariants;
-  isNavLink?: boolean;
-  isCurrentPage?: boolean;
-  target?: string;
-  rel?: string;
-  className?: string;
-}
+import { ForwardedRef, forwardRef, useState, useEffect } from 'react';
+import { NavLink, Link as RouterLink } from 'react-router-dom';
+import { LinkProps } from './Link.types';
 
 const Link = forwardRef(
   (
     { variant, href, to, isNavLink, isCurrentPage, className, children, target, rel, ...linkAttrs }: LinkProps,
     ref: ForwardedRef<HTMLAnchorElement>
   ): JSX.Element => {
-    let classes = '';
-    switch (variant) {
-      case 'cta':
-        classes = 'bsds-link-cta';
-        break;
-      case 'ghost':
-        classes = 'bsds-link-ghost';
-        break;
-      case 'mailto':
-        classes = 'bsds-link-cta bsds-link-mailto';
-        break;
-      case 'nav':
-        classes = 'bsds-link-nav';
-        break;
-      case 'subtle':
-        classes = 'bsds-link-subtle';
-        break;
-      default:
-        classes = 'bsds-link';
-        break;
+    let baseClass = 'bsds-link';
+
+    if (variant) {
+      /** CTA variants are styled to look like a button */
+      const isCtaVariant = variant.startsWith('cta');
+      baseClass += isCtaVariant ? `-cta ${baseClass}-${variant}` : `-${variant}`;
```

Comment:
@kaseyllabelle @APJohns Please let me know if you oppose this change. It goes along with the change to the style file here: https://github.com/bos-sci/anatomy-react/pull/90/files#diff-4b631f415e99d7a7d6b4647cc1fd36a357daa4646b8ff528b4a00c6decaeab57

I ran tests and checked Storybook to confirm it all works the same. "cta" variants look like buttons, and this streamlines adding new ones to the stylesheet.

![link-vars](https://github.com/bos-sci/anatomy-react/assets/96078669/2cd4314d-1340-438d-b7a0-5b607f682fea)